x-pack/test/functional/services/cases/create.ts

```diff
@@ -54,11 +55,13 @@ export function CasesCreateViewServiceProvider(
       description = 'desc' + uuidv4(),
       tag = 'tagme',
       severity = CaseSeverity.LOW,
+      category,
       owner,
     }: CreateCaseParams) {
       await this.setTitle(title);
       await this.setDescription(description);
       await this.setTags(tag);
+      await this.setCategory(category);
```

Comment:
nit: let's first if the category is defined before setting it. Like the severity.